test/suite/test_assert01.py

```diff
@@ -39,15 +39,15 @@ def timestamp_str(t):
 class test_assert01(wttest.WiredTigerTestCase, suite_subprocess):
     base = 'assert01'
     base_uri = 'file:' + base
-    uri_always = base_uri + '.always.wt'
+    uri_all = base_uri + '.all.wt'
     uri_def = base_uri + '.def.wt'
-    uri_never = base_uri + '.never.wt'
```

Comment:
Never is one of the options, why are we removing it?

Reply:
Probably because it's already tested with `cfg_diagnostic`.

@kommiharibabu Was there a different reason?

Reply:
Yes, that's correct. `never` was an option at first, so I updated all the tests.